### PR TITLE
Disable tabs by network configuration

### LINKED ISSUE #1721: Hide unsupported tabs by network
### Description of the Feature

It should be possible to disable or hide some tabs that doesn't make sense in a selected network.

### PR DESCRIPTION
**Pull request by alepc253**
Closes #1721

### Description

Gives the possibility of disable navigation tabs that don't make sense in a certain network.

### Changes

* Addition of new parameter `unsupportedTabs` in network configuration. 
* Addition of new prop `isNotEnabled` in NavigationLink (this prop determines the rendering of the component).

### Steps to Test

1. Choose a network config and add, for example, `unsupportedTabs: [TAB.ENS, TAB.SWAP]` in file `staticNetworks.ts`
2. Select that network
3. Check visually that this tabs are not displayed

### CODE REVIEW

**Comment by alepc253** [comment]
@dternyak @HenryNguyen5 let me know if we should change something else

**Comment by dternyak** [comment]
@alepc253 Looks like we've got a few conflicts here that should be fixed before merge!

**Comment by alepc253** [comment]
@dternyak yep, I tried to update the PR but it's getting difficult (develop is changing). Actually, the real changes for this functionality are just a few lines in a couple of files... Tell me if I can help with this. Thanks!

**Comment by alepc253** [comment]
@dternyak I fixed again those conflicts... the only one is in a file I changed (and don't understand why is marked as a conflict)

File: webpack_config/devServerHTTPS.js [hunk 1]
```diff
@@ -0,0 +1 @@
+npm run dev:https
```
Comment (dternyak) [inline webpack_config/devServerHTTPS.js]: What is this file for?

Comment (alepc253) [reply webpack_config/devServerHTTPS.js]: @dternyak I'm not sure... ignore it if it makes no sense for you (I guess I might have added it long time ago trying to run the site as https and I kept it by mistake)

Comment (dternyak) [reply webpack_config/devServerHTTPS.js]: Got it. Do you mind removing this file in that case?

**Comment by alepc253** [comment]
@dternyak done! thanks

File: webpack_config/devServerHTTPS.js (hunk 1 above)
Comment (alepc253) [reply webpack_config/devServerHTTPS.js]: Done

**Pull request merged**